python/paddle/fluid/layers/detection.py

```diff
@@ -909,3 +910,95 @@ def _is_list_or_tuple_and_equal(data, length, err_info):
     box.stop_gradient = True
     var.stop_gradient = True
     return mbox_locs_concat, mbox_confs_concat, box, var
+
+
+def anchor_generator(input,
+                     anchor_sizes=None,
+                     aspect_ratios=None,
+                     variance=[0.1, 0.1, 0.2, 0.2],
+                     stride=None,
+                     offset=0.5,
+                     name=None):
+    """
+    **Anchor generator operator**
+
+    Generate prior boxes for SSD(Single Shot MultiBox Detector) algorithm.
+    Each position of the input produce N prior boxes, N is determined by
+    the count of min_sizes, max_sizes and aspect_ratios, The size of the
+    box is in range(min_size, max_size) interval, which is generated in
+    sequence according to the aspect_ratios.
+
+    Args:
+       input(Variable): The Input feature map, the format is NCHW.
+       anchor_sizes(list|tuple|float value): Anchor sizes of generated anchors.
+       Defalut: [64, 128, 256, 512]
```

Comment:
list|tuple|float value -> list|tuple|float

There is no default value in the interface. And need to explain what is anchor size?

Reply:
revised